conn_linux_test.go

```diff
@@ -0,0 +1,96 @@
+//go:build !freebsd
```

Comment:
this is a using a Linux-only syscall - we also have other non-Linux platforms besides FreeBSD that are supported (e.g. Darwin), please adjust accordingly

Reply:
I see, adjusted accordingly, please recheck.

Reply:
you can just add `linux` as a build tag 😄 but I only now just noticed that you already put this in a file with a `_linux` suffix, which implicitly adds the corresponding build tag, so just removing the comments should be fine

Reply:
thanks for the input, speaking using sledgehammer to crack a nut :)